0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -60,11 +64,12 @@ spec](https://github.com/interledger/rfcs/blob/master/asn1/BilateralTransferProt
   ASN.1 spec.
 
 - A **Sub-Protocol** is a protocol which isn't defined by BTP and is carried
-  in the protocol data (see below).
+  in the protocol data (see below). The first one is the primary sub-protocol,
+  subsequent entries are secondary sub-protocols.
```

Comment:
It should say something about the relationship between primary and secondary sub-proto. A secondary sub-protocol adds additional information to a primary-subprotocol.

Also, we should say that unrelated sub-protocols SHOULD NOT go into the same packet, but be sent in separate packets.

Reply:
Also:
> If the packet contains more than one protocol with the same name, reject the whole packet with a BTP error

Reply:
I think the exact semantics of sub-protocols belongs in the format section rather than the terminology section.

Reply:
Can we please put some examples of what we mean by this? The concept of a sub-protocol is already a bit confusing, and now it seems even more so because of this primary vs secondary distinction.